src/encoded/types/dataset.py

```diff
@@ -302,7 +302,6 @@ def default_analysis(self, request, status, analyses):
             # We want highest version, date, etc. so reverse sort order. Access value from top item.
             return sorted(analysis_ranking.items(), reverse=True)[0][1]
 
-
 def convert_date_string(date_string):
```

Comment:
Seems like we’re encouraging [two blank lines](https://github.com/ENCODE-DCC/encoded/pull/3776#pullrequestreview-615728952) between functions.